reana-ui/src/util.js

```diff
@@ -35,7 +35,7 @@ export const statusMapping = {
  */
 export const healthMapping = {
   healthy: "green",
-  warning: "orange",
+  warning: "brown",
```

Comment:
Why do we need to change this from orange to brown?

Reply:
> I took the liberty to change it to brown to make it more in line with other colors used inside profile page. Since we introduced notifications there which have brown (warning) and red (error) colors, keeping also the orange there seemed a bit too much. Now this brown color match the warning notification color (you can see the 70% label and disk usage warning in the image bellow).

:+1:

> But indeed there is inconsistency between what we get from the server and what we display. E.g. Critical level is >= 85 percent and we display it as a warning even though the pie chart label color is red

We can continue this discussion in the [other thread](https://github.com/reanahub/reana-ui/pull/275#discussion_r892446483)!